Makefile

```diff
@@ -133,7 +133,7 @@ tf: tfp
 	$(EXEC) vendor/bin/phpunit --group functional || true
 
 tfp:            ## Prepare the PHP functional tests
-tfp: vendor assets-amp assets-prod
+tfp: vendor assets-amp assets-prod up
```

Comment:
Please remove this. It should be done only once while `tfp` task can be done a lot of time per day.